guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-satellite-server.adoc

```diff
@@ -66,13 +73,18 @@ To update the certificates on a currently running Katello installation, run:
     --certs-update-server --certs-update-server-ca
 ----
 endif::[]
-
++
+Note that you must not access or modify `/root/ssl-build`.s
```

Comment:
```suggestion
Do not access or modify the contents of `/root/ssl-build`.
```

Reply:
Also, you could consider making this [IMPORTANT].

Reply:
I had considered it but then we will have 3 important notes within the same chapter which I guess, is not good for a readability perspective. There are already 2.